pywbemtools/pywbemcli/_cmd_server.py

```diff
@@ -375,7 +375,7 @@ def cmd_server_centralinsts(context, options):
                     scoping_class=options['scoping_class'],
                     scoping_path=options['scoping_path'],
                     reference_direction=options['reference_direction'])
-                row.append(":".join([str(p) for p in ci]))
+                row.append("/n".join([str(p) for p in ci]))
```

Comment:
This change fixes issue #363. Either say that in commit message title and description, or move to a separate PR.

Reply:
My BooBoo.  It was not supposed to.  I accidently leaked one pr into another. I am taking this one out and making it separate.

Reply:
REMOVED.